### DISCUSSION TITLE
Feature to manage "projects" in application.

### DISCUSSION THREAD

**Discussion by kasipavankumar** [community/community#4104] [Mobile] [answered]
It would be really handy and useful to manage projects right from the mobile application.

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to manage projects internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

And please, upvote the existing feature request for [Projects here](https://github.com/github/feedback/discussions/3023).

**Reply by kasipavankumar** [reply]
Thank you. Looking forward for the feature.

**Comment by candyho** [comment]
Hey👋just want to inform you we've just shipped [view and edit projects(beta) custom fields]( https://github.blog/changelog/2022-05-09-view-and-edit-projectsbeta-custom-fields-on-github-mobile/) to both iOS and Android last week! You can now edit custom text, number, date, and single-select fields for issues and pull requests that belong to the new projects experience from within the triage sheet. Try it out and let us know what you think! Stay tuned for more updates!

**Reply by kasipavankumar** [reply]
Awesome! Will try out the feature and give feedback.
